src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx

```diff
@@ -39,48 +71,30 @@ const Contacts = () => {
       <section className="profile-info-card">
         <div className="row vads-u-border-color--gray-lighter vads-u-color-gray-dark vads-u-display--flex vads-u-flex-direction--column vads-u-padding-x--2 vads-u-padding-y--1p5 medium-screen:vads-u-padding--4 vads-u-border--1px">
           <h2 className="vads-u-font-family--sans vads-u-font-size--base vads-u-margin--0">
-            Medical emergency contact
+            Medical emergency contacts
           </h2>
           <p className="vads-u-color--gray-medium vads-u-margin-top--1 vads-u-margin-bottom--1">
-            The person we’ll contact in an emergency.
+            The people we’ll contact in an emergency.
           </p>
-          {emergencyContact && (
-            <Contact
-              testId="phcc-emergency-contact"
-              key={emergencyContact.id}
-              {...emergencyContact.attributes}
-            />
-          )}
-          {!emergencyContact && (
-            <div>
-              To add an emergency contact, call us at <HelpDeskContact />.
-            </div>
-          )}
+          {renderEmergencyContacts}
```

Comment:
How does this look without any contacts? Is that possible either here or nok?

Reply:
Instructions are rendered if no EC or no NOK